examples/offlimb/radial_gradient_filters.py

```diff
@@ -0,0 +1,58 @@
+"""
```

Comment:
Move the example file one folder higher.
For now just put them in `examples/` and no subfolders.

Reply:
Done